exec.ts

```diff
@@ -0,0 +1,34 @@
+import querystring = require('querystring');
+import stream = require('stream');
+
+import { WebSocketHandler } from './web-socket-handler';
+import { KubeConfig } from '@kubernetes/typescript-node';
+
+export class Exec {
+    'handler': WebSocketHandler;
+
+    public constructor(config: KubeConfig) {
+        this.handler = new WebSocketHandler(config);
+    }        
+
+    // TODO: make command an array and support multiple args
```

Comment:
nit: I've created https://github.com/kubernetes-client/typescript/issues/28 to track this